### ISSUE TITLE
Custom rule to hide/remove a language

### ISSUE THREAD

**Issue by Onegenimasu** [wakatime/vscode-wakatime#311]
Not sure if this is the best place to post this, but oh well.

My coding activity recently got littered with languages like "Text", "AsciiDoc", "Markdown", "Git Config" etc. and I wanted to remove them from my coding activity. I attempted to set up a rule, but couldn't find how to just hide a language - I tried setting it to "Ignore List", which appeared from the dropdown menu, but they just merged to one *language* called "Ignore List", and I'm not sure how to deal with it now.

So in short: Is it possible to make a rule for wakatime to hide/ignore a language from activity?
If not, consider this issue a feature request.

Thank you.

**Comment by alanhamlett** [comment]
It's only possible to hide/exclude activity on the plugins, not on the website. Once it's on your dashboard on the website you can delete it with a custom rule:

`If language equals X then change project to DELETE`

Then once the rule finishes applying to past activity, visit https://wakatime.com/projects/DELETE and use the dropdown menu a the top-right of the page to delete that project and all it's activity.

To ignore that language in the future, you need to match it's file extension using an exclude regex in your `~/.wakatime.cfg` file:
https://wakatime.com/faq#sensitive-files
